src/tasks/weapondamage.ts

```diff
@@ -270,6 +270,16 @@ export const WeaponDamageQuest: Quest = {
         $effects`Spit Upon, Pyramid Power`.forEach((ef) => {
           if (CommunityService.WeaponDamage.actualCost() >= 5) wishFor(ef); // The effects each save 2 turns on spelltest as well
         });
+
+        if (
+          CommunityService.WeaponDamage.actualCost() >= 5 &&
+          !get("_cargoPocketEmptied") &&
+          have($item`Cargo Cultist Shorts`) &&
+          !get("instant_saveCargoShorts", false)
+        ) {
+          visitUrl("inventory.php?action=pocket");
+          visitUrl("choice.php?whichchoice=1420&option=1&pocket=284");
+        }
```

Comment:
If we are doing this here we need to remove the corresponding weapon damage task (else that would always trigger first before we do this check here)

Also, considering that a cargo pull is worth less than a wish, we should probably
(1) Shift this logic above the `wishFor()` segment, and
(2) Only require an `actualCost() > 1` check (similar to that of the offhand remarkable segment)

Reply:
That's fair. There's probably some better (but more complicated) logic that we can use here, but this is sufficient for now.